src/core/Akka.Streams/Dsl/Restart.cs

```diff
@@ -494,7 +494,7 @@ protected SubSinkInlet<TOut> CreateSubInlet(Outlet<TOut> outlet)
                         Fail(Out, ex);
                     else
                     {
-                        Log.Warning($"Restarting graph due to failure. Stacktrace: {ex.StackTrace}");
+                        Log.Warning("Restarting graph due to failure: {0}", ex);
```

Comment:
The `Warning` method can now take the `Exception` as an input parameter. Please change to use that method.